politeiawww/api/www/v2/v2.go

```diff
@@ -168,3 +169,9 @@ type VoteDetailsReply struct {
 	EndBlockHeight   uint32   `json:"endblockheight"`   // Height of vote end
 	EligibleTickets  []string `json:"eligibletickets"`  // Valid voting ticket
 }
+
+// QuiesceReply is the reply to the Quiesce command. It includes the quiesce
+// toggle value
+type QuiesceReply struct {
```

Comment:
There should be a `type Quiesce struct{}` even if its empty so that it is explicit.